frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts

```diff
@@ -0,0 +1,96 @@
+import { timeouts, testName, submitButton, errorMessage } from '../support/utils';
+
+const testDetailsPage = (pageTitle: string, sectionHeading: string, testLabel = true) => {
+  cy.getTestElement('resource-title', timeouts.pageLoad).contains(pageTitle); // long timeout for detail page to load
+  cy.getTestElement('section-heading')
+    .first()
+    .should('have.text', sectionHeading);
+  if (testLabel) {
+    cy.get('.co-m-label')
+      .first()
+      .contains(`alertname=${pageTitle}`);
+  }
+};
+
+describe('Monitoring: Alerts', () => {
+  // TODO: need to move these to before/after all specs run,
+  //  not just before/after this spec
+  before(() => {
+    cy.createTestProject(testName);
+  });
+  after(() => {
+    cy.deleteTestProject(testName);
+  });
+
+  it('displays the Alerts list page', () => {
+    cy.clickNavLink(['Monitoring', 'Alerting']);
+    cy.getTestElement('resource-title').should('have.text', 'Alerting');
+  });
+
+  it('does not have a namespace dropdown', () => {
```

Comment:
So like we are discussing in slack, I think each `it()` should be its own atomic test, per best practices.  So they should all likely start with `cy.get(the-page)`, and work from there.

Its ok if one `it()` does several asserts.  

I think we can model these:

```
it('should ....') {
   - visit the page
   - do a task
   - assert 1-n related things things
   - perhaps do another task
   - assert 1-n related things things
}
```
We can still have several `it()` blocks of related bits, so long as each starts fresh.

Reply:
fixed, each `it()` is atomic.